02-clients/01-studio.md

```diff
@@ -0,0 +1,421 @@
+---
+pageTitle: TypeDB Studio
+keywords: typedb, client, studio
+longTailKeywords: typedb studio, typedb studio preferences, typedb studio connection
+summary: Brief intro for TypeDB Studio.
+toc: false
+---
+
+# TypeDB Studio
+
+TypeDB Studio is an IDE specifically designed to work with 
+[TypeQL]() query language and TypeDB databases.
+
+It has unique set of functions and abilities to help with development using TypeDB:
+
+- Friendly graphical user interface (GUI) to design schemas and queries for TypeDB databases.
+- Syntax highlighting for TypeQL.
+- Connection to TypeDB servers, including handling transactions.
+- Types explorer for a TypeDB database.
+- Query execution, including schema and data, read and write transactions.
+- Local query validation with constraints of an actual database schema before sending the query to server.
+- Graph visualization for query responses.
+
+## Installation
+
+### Requirements
+
+TypeDB Studio do not require any additional packages to be installed prior the installation, but it needs a TypeDB 
+server to perform most it's functions. We can use either remote or local TypeDB server.
+
+<div class="note">
+[Note]
+The default TypeDB settings are to run server listening for `localhost` connections with port number `1729`.
+</div>
+
+<div class="note">
+[Important]
+We recommend using the latest version of TypeDB server and a [compatible](#version-compatibility) version of TypeDB 
+Studio. 
+</div>
+
+### Download & install
+
+Download TypeDB distributive/package for any popular OS family from the
+[Download page](https://vaticle.com/download#typedb-studio. 
+
+Follow proper installation procedure of the OS in use.
+
+<div class="note">
+[Important]
+Make sure to grant access for installation wizard to install TypeDB Studio in case any security prompts will appear.
+</div>
+
+<div class="note">
+[Note]
+For **macOS** TypeDB Studio can be installed via homebrew:
+
+```
+brew tap vaticle/tap
+brew install --cask vaticle/tap/typedb-studio
+```
+</div>
+
+## Prepare for work
+
+Right after starting TypeDB Studio we can see its main window as it is shown below.
+
+![TypeDB Studio GUI](../images/studio/studio-gui.png)
```

Comment:
I think this image could use a little work. Some suggestions:
* It would be great if we used our default font (Titillium Web) for the numbers: https://fonts.google.com/specimen/Titillium+Web
* We should try putting boxes around the sections of Studio we're highlighting, preferably with rounded corners.
* Let's use our light green accent (#02DAC9) for the numbers and the boxes rather than the current red.

Reply:
Likewise for the rest of the places we've added highlights (arrows, boxes, circles, etc).

Reply:
Implemented with the font, but the Pink color from our new color palette. Haikal chose Pink.

Reply:
I love it! Looks much better.